src/NumericalAlgorithms/Spectral/Projection.hpp

```diff
@@ -14,20 +14,60 @@ class Mesh;
 
 namespace Spectral {
 
+/// The portion of an element covered by a child element.
+enum class ChildSize { Full, UpperHalf, LowerHalf };
+
 /// The portion of an element covered by a mortar.
-enum class MortarSize { Full, UpperHalf, LowerHalf };
+using MortarSize = ChildSize;
 
-std::ostream& operator<<(std::ostream& os, MortarSize mortar_size) noexcept;
+std::ostream& operator<<(std::ostream& os, ChildSize mortar_size) noexcept;
 
 /*!
- * \brief The projection matrix from a 1D mortar to an element.
+ * \brief The projection matrix from a child element to its parent.
  *
- * \details
  * The projection matrices returned by this function (and by
- * projection_matrix_element_to_mortar()) define orthogonal projection
- * operators between the spaces of functions on the boundary of the
- * element and the mortar.  These projections are usually the correct
- * way to transfer data onto and off of the mortars.
+ * projection_matrix_parent_to_child()) define orthogonal projection operators
+ * between the spaces of functions on a parent element and its children. These
+ * projections are usually the correct way to transfer data between elements in
+ * a mesh-refinement hierarchy, as well as between an element face and its
+ * adjacent mortars.
+ *
+ * These functions assume that the `child_mesh` is at least as fine as the
+ * `parent_mesh`, i.e. functions on the `parent_mesh` can be represented exactly
+ * on the `child_mesh`. In practice this means that functions can be projected
+ * to a mortar (the `child_mesh`) from both adjacent element faces (the
+ * `parent_mesh`) without losing accuracy. Similarly, functions in a
+ * mesh-refinement hierarchy don't lose accuracy when an element is split
+ * (h-refined). For this reason, the `projection_matrix_child_to_parent` is
+ * sometimes referred to as a "restriction operator" and the
+ * `projection_matrix_parent_to_child` as a "prolongation operator".
+ *
+ * The projections implemented here are \f$L^2\f$ projections, or "Galerkin"
+ * projections. They are defined such that the projection from a parent to one
+ * of its children ("prolongation") is an (exact) interpolation, and the
```

Comment:
I get a bunch of errors like: `Projection.hpp:47: warning: Found unknown command '@_fakenlR'` when trying to build the docs